pkgs/development/python-modules/mizani/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, matplotlib
+, palettable
+, pandas
+, pytest-cov
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mizani";
+  version = "0.7.3";
+
+  src = fetchFromGitHub {
+    owner = "has2k1";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "04r53dp5jbklv8l9ncgc5wiq0gx25y73h65gmmbbfkxwgsl3w78l";
+  };
+
+  propagatedBuildInputs = [ matplotlib palettable pandas ];
+
+  checkInputs = [ pytest-cov pytestCheckHook ];
```

Comment:
Please patch out `pytest-cov` and remove the input. This doesn't add value for us.

Something like this should do the trick:

```bash
  postPatch = ''
    substituteInPlace pytest.ini \
      --replace " --cov=mizani --cov-report=xml" ""
  '';
```

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/137458/commits/7dae4c8f28e2616823a6edd826e47128afa526f3.